unic/ucd/normal/src/canonical_combining_class.rs

```diff
@@ -53,7 +53,7 @@ impl TotalCharProperty for CanonicalCombiningClass {
 impl NumericCharProperty<u8> for CanonicalCombiningClass {
     /// Get numeric value for character property value
     fn number(&self) -> u8 {
-        self.number()
+        Self::number(self)
```

Comment:
Wait. How does this work? Is this not self-recursive and should it not just be `self.0`?